dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp

```diff
@@ -319,7 +315,13 @@ void DAGStorageInterpreter::executeImpl(DAGPipeline & pipeline)
         // and ask RN to send requests again with correct region info. When RN updates region info,
         // RN may be sending requests to other WN.
 
-        throw Exception("Rejected disaggregated DAG execute because RN region info does not match", DB::ErrorCodes::REGION_EPOCH_NOT_MATCH);
+        RegionException::UnavailableRegions region_ids;
+        for (const auto & info : context.getDAGContext()->retry_regions)
+            region_ids.insert(info.region_id);
```

Comment:
should also add the region_ids from `DAGStorageInterpreter::region_retry_from_local_region`

Reply:
Seems that region_retry_from_local_region has already been handled by `buildRemoteRequests`:

```c++
// Build remote requests from `region_retry_from_local_region` and `table_regions_info.remote_regions`
std::vector<RemoteRequest> DAGStorageInterpreter::buildRemoteRequests(const DM::ScanContextPtr & scan_context)
{
...
        for (const auto & r : table_regions_info.remote_regions)
            retry_regions_map[physical_table_id].emplace_back(std::cref(r));
...
    for (auto & r : region_retry_from_local_region)
    {
        retry_regions_map[region_id_to_table_id_map[r.get().region_id]].emplace_back(r);
...
...
        const auto & retry_regions = retry_regions_map[physical_table_id];
        for (const auto & r : retry_regions)
            context.getDAGContext()->retry_regions.push_back(r.get());
...
}
```